CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Change Log の形式は [Keep a Changelog](http://keepachangelog.com/) に従い
 
 ### Added
 
+- Add csv to gemspec for new Ruby versions
```

Comment:
Please align with the format of other Changelogs.

Reply:
https://github.com/yamat47/japanese_address_parser/pull/108/commits/03a8964af26bde8406bc7e2d291c8cc7aeee45a3